### PR TITLE
Imports Validation Schemas into the Examples README during CI

### PR DESCRIPTION
**Pull request by JBBianchi**
**Closed because of messy commit history, see #805 instead.**

### CODE REVIEW

**Review by cdavernas — APPROVED** [review]
LGTM! Many thanks ❤️

**Comment by JBBianchi** [comment]
I forgot to signoff my commits... If necessary, I created an [other branch](https://github.com/JBBianchi/specification/tree/feat-examples-readme-templating) with a signed off commit. I can close this PR and open a new one if it's a big no go.

@cdavernas @ricardozanini  @tsurdilo @antmendoza WDYT ?

**Comment by ricardozanini** [comment]
@JBBianchi I think you can amend your commit and add the `-s` :D

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@JBBianchi let's discuss tomorrow about the planning and the steps to install the bot! We might do it tomorrow during the meeting if necessary: #804

File: .ci/examples-readme-hydration/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "examples-readme-hydration",
+  "version": "0.1.0",
+  "description": "Builds ./examples/README.md based on ./examples/README_TEMPLATE.md",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "node ./index.js"
+  },
+  "keywords": ["cncf", "serverless", "workflow", "specification"],
+  "author": "CNCF Serverless Workflow Specification",
+  "license": "ISC",
```
[the remark this answers is not in the document]
Comment (JBBianchi) [reply .ci/examples-readme-hydration/package.json:11]: Sure, it's the default when generating the npm project I think. As far as I recall, I copy/pasted from https://github.com/serverlessworkflow/specification/blob/main/.ci/validation/package.json

**Comment by JBBianchi** [comment]
> @JBBianchi I think you can amend your commit and add the `-s` :D

I just did but I think I messed up ^^'

I'll just close and open a new PR, it feels safer.

**Pull request closed without merging**